### ISSUE TITLE
[Android] Unable to compile v2.2.3

### ISSUE THREAD

**Issue by chancezeus** [OneSignal/OneSignal-Cordova-SDK#277]
When running cordova build android I am presented with the following error after the most recent update:

```
Could not compile script '/home/mark/Documents/Werk/APPelit/motorhit/cordova/platforms/android/onesignal-cordova-plugin/motorhit-build-extras-onesignal.gradle'.
> startup failed:
  script '/home/mark/Documents/Werk/APPelit/motorhit/cordova/platforms/android/onesignal-cordova-plugin/motorhit-build-extras-onesignal.gradle': 1: Only Project build scripts can contain plugins {} blocks
  
  See https://docs.gradle.org/4.1/userguide/plugins.html#sec:plugins_block for information on the plugins {} block
  
   @ line 1, column 1.
     plugins {
     ^
  
  1 error
```

It seems that the plugin's gradle config is having some incorrect configuration.

**Comment by jkasten2** [comment]
@chancezeus We didn't run into the issue on the first pass of testing. However we are now able to reproduce the issue. You can rollback to 2.2.2 by running the following until we resolve the issue.
```bash
cordova plugin rm onesignal-cordova-plugin
cordova plugin add onesignal-cordova-plugin@2.2.2
```

**Comment by chancezeus** [comment]
Already did, just wanted to report it before/in case anyone else runs into it...

**Comment by jkasten2** [comment]
@chancezeus Thanks again for reporting. Just released 2.2.4 to npm to resolve this issue.
